apax/nodes/analysis.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import pathlib
+
+import ase.io
+import znflow
+import zntrack.utils
+from ipsuite import fields
+
```

Comment:
if you import IPSuite here, you could use the base class as well.

Reply:
fair, I'll switch to the base class

Reply:
ah yeah this is also why the tests fail, my bad